source/lex.h

```diff
@@ -1284,54 +1472,80 @@ auto lex_line(
 
                 //G string-literal:
                 //G     encoding-prefix? '"' s-char-seq? '"'
+                //G     encoding-prefix? 'R"' d-char-seq? '(' s-char-seq? ')' d-char-seq? '"'
```

Comment:
Shouldn't this be prefixed with `'$'?`? Or is that somehow implied by the apparently unused `token` grammar?
```Cpp2
//G token:
//G     identifier
//G     keyword
//G     literal
//G     operator-or-punctuator
//G
//G operator-or-punctuator:
//G     operator
//G     punctuator

//G punctuator: one of
//G     '...' '.'
//G     '::' ':'
//G     '{' '}' '(' ')' '[' ']' ';' ',' '?' '$'
```

Reply:
Actually, see https://github.com/hsutter/cppfront/issues/387#string-literal.